test/test_google_geocoder.rb

```diff
@@ -132,6 +132,13 @@ def test_google_administrative_area_level_3
      end
    end
 
+  def test_google_city_improved_ordering
+    VCR.use_cassette('google_city_ordering') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.